langfuse/openai.py

```diff
@@ -325,6 +330,8 @@ async def _get_langfuse_data_from_async_streaming_response(
         resource, responses
     )
 
+    langfuse.trace(id=generation.trace_id, output=completion)
```

Comment:
In these cases, we do not know anymore whether the trace was generated by the user becore and whether he provided the traceId himself. Can we somehow get that information here and only update in this case? Similar to how we do that with the input.